src/components/TableHeadCell.tsx

```diff
@@ -105,9 +106,9 @@ class TableHeadCell extends React.Component {
             placement={'bottom-start'}
             classes={{
               tooltip: classes.tooltip,
+              popper: classes.mypopper
             }}
             enterDelay={300}
-            classes={{ popper: classes.mypopper }}
```

Comment:
Here was an example where there were two definitions of `classes` which the TS compiler picked up (note the same issue further down in this file).  I have combined the two classes, although this may have unforeseen consequences as I don't know how JS handled the issue previously.  Perhaps only the latter or former class was applied.